scripts/ioc_deploy.py

```diff
@@ -313,6 +322,32 @@ def get_version() -> str:
         return "unknown.dev"
 
 
+def _clone(
+    name: str,
+    github_org: str,
+    release: str = "",
+    working_dir: str = "",
+    target_dir: str = "",
+    verbose: bool = False,
+) -> subprocess.CompletedProcess:
+    """
+    Clone the repo or raise a subprocess.CalledProcessError
+    """
+    cmd = ["git", "clone", f"git@github.com:{github_org}/{name}", "--depth", "1"]
+    if release:
+        cmd.extend(["-b", release])
+    if target_dir:
+        cmd.append(target_dir)
+    kwds = {"check": True}
+    if working_dir:
+        kwds["cwd"] = working_dir
+    if not verbose:
+        kwds["stdin"] = subprocess.PIPE
```

Comment:
Why are we redirecting stdin here?  Do we mean to redirect stdout?

Reply:
whoops, yes
I guess all the important stuff to redirect was in stderr because I didn't notice anything wrong